releases/stellar/2025-01-ITS-v1.0.0.md

```diff
@@ -106,7 +68,7 @@ ITS hub contract configuration in json file must include the following attribute
     "InterchainTokenService": {
       ...
       \"$CHAIN\": {
-        "maxUintBits": 64,
+        "maxUintBits": 128,
         "maxDecimalsWhenTruncating": 7
```

Comment:
```suggestion
        "maxDecimalsWhenTruncating": 255
```
see the decimals type our interchain token uses. `u128` is big enough for any practical decimal values, so no truncation needs to be done